### DISCUSSION TITLE
Accent characters not displayed correctly

### DISCUSSION THREAD

**Discussion by jdholtz** [cmus/cmus#1315] [Troubleshooting] [answered]
Hello. When I have an MP3 file that has an artist or song title id3 tag containing an accent, it is displayed very incorrectly (picture below). I checked out #743, but that didn't seem to be the exact same issue I am having. I am currently using the official cmus package from Arch Linux.

Here is how to tag an mp3 file with an artist containing an accent: `id3tag -a "Accént" file.mp3`. This is also an issue with titles. What is the solution to this to have accented characters display correctly? Thanks!
![image](https://github.com/cmus/cmus/assets/89617856/4b6f8050-61ee-40e1-a7e3-d77f44cd6533)

**Comment by gavtroy** [comment, marked as answer]
I think it's `id3tag` that doesn't like them. Substitute `mid3v2` or another tagging tool and they work fine.

**Reply by jdholtz** [reply]
Did not know about mid3v2 until now and it worked once I converted it. Thanks!
